core/services/pipeline/task.bridge.go

```diff
@@ -91,11 +95,16 @@ func (t *BridgeTask) Run(ctx context.Context, vars Vars, inputs []Result) Result
 		"url", url.String(),
 	)
 
-	responseBytes, elapsed, err := makeHTTPRequest(ctx, "POST", URLParam(url), requestData, allowUnrestrictedNetworkAccess, t.config)
+	responseBytes, headers, elapsed, err := makeHTTPRequest(ctx, "POST", URLParam(url), requestData, allowUnrestrictedNetworkAccess, t.config)
```

Comment:
Perhaps a `httpResponse` struct might be nice?

Reply:
I'll do that in a follow-up PR.